packages/js/src/executors/tsc/tsc.impl.ts

```diff
@@ -94,7 +93,8 @@ export async function* tscExecutor(
 
   return yield* eachValueFrom(
     compileTypeScriptFiles(options, context, async () => {
-      await updatePackageAndCopyAssets(options, projectRoot);
+      await assetHandler.processAllAssetsOnce();
+      updatePackageJson(options.main, options.outputPath, projectRoot);
```

Comment:
after this change, git ignored files (e.g., files under `node_modules/..`) are no longer copied.